CODE_OF_CONDUCT.md

```diff
@@ -59,8 +59,8 @@ representative at an online or offline event.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported to Brent Yorgey at [byorgey@gmail.com](mailto:byorgey@gmail.com), or 
-`byorgey` on the Libera.Chat IRC network. All complaints will be reviewed and 
+reported to Brent Yorgey at [byorgey@gmail.com](mailto:byorgey@gmail.com), or
+`byorgey` on the Libera.Chat IRC network. All complaints will be reviewed and
```

Comment:
I did not update this on purpose: this is about a way to contact me personally, and email or IRC remain some of the best ways to contact me.